apps/examples/elf/wifiapp/wifiapp.c

```diff
@@ -42,6 +42,146 @@ int main(int argc, char **argv)
 	fprintf(stdout, "%15s: Hello, World!! on stdout\n", argv[0]);
 	fprintf(stderr, "%15s: Hello, World!! on stderr\n", argv[0]);
 
+#ifdef CONFIG_EXAMPLES_MPU_TEST
+	volatile uint32_t *address;
+	char ch;
+
+	printf("\nPress R - For Read Test\n");
+	printf("Press W - For Write Test\n");
+
+	ch = getchar();
+	if (ch == 'R' || ch == 'r') {
+		printf("\nPress A for app MPU test\n");
+		printf("Press K for kernel MPU test\n");
+
+		ch = getchar();
+
+		if (ch == 'A' || ch == 'a') {
+			address = 0x20250000;
```

Comment:
This value can be changed with based board.
Please make it general.

Reply:
It is difficult to make this general, because the wifiapp is built as a different application and it does not have the idea about start address of kernel and micomapp. 

However, I can add preprocessor definition in the wifiapp like below and it can be changed based on the board. What do you think?

#ifdef ARCH_IMXRT
#define KERNEL_START_ADDR    0x60000000
#define APP_START_ADDR          0x20200000
...
...
#endif

Reply:
Yes. I think it is good to me.

Reply:
OK. I fixed it.